tests/lib/eslint/eslint.js

```diff
@@ -918,6 +918,112 @@ describe("ESLint", () => {
                 assert.strictEqual(results[0].messages[0].ruleId, "no-undef");
             });
         });
+
+        describe("TypeScript config files", () => {
```

Comment:
[Testing] What do you think about including a test file that includes the not-very-friendly-to-transpilation TypeScript features?

* `const enum`
* `namespace`
* `@` `experimentalDecorators`

Reply:
Interesting, I personally would never use them because they are not transpilation friendly 😇 But good points - I will try to do a matrix tests in importx first to see how each solution works with them